pkg/scheduler/factory/factory.go

```diff
@@ -177,6 +177,7 @@ func NewConfigFactory(args *ConfigFactoryArgs) scheduler.Configurator {
 		client:                         args.Client,
 		podLister:                      schedulerCache,
 		podQueue:                       core.NewSchedulingQueue(),
+		nodeLister:                     args.NodeInformer.Lister(),
```

Comment:
this is not that necessary, but just to keep the same pattern - init all listers in &configFactory{}, then add watch functions afterwards.